### PR TITLE
About page

### PR DESCRIPTION
**Pull request by sohilpandya**
Please review after #32. 

Added About page, this one is responsive.

### CODE REVIEW

File: src/components/Footer.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          <p className="ma0 lh-copy">70 Cowcross Street</p>
+          <p className="ma0 lh-copy">London, EC1M 6EJ</p>
+        </section>
+      </section>
+
+      <section className="w-33 pa2 tc flex justify-center">
+        <div className="mh2"><Twitter/></div>
+        <div className="mh2"><Fb/></div>
+        <div className="mh2"><Instagram/></div>
+      </section>
```
Comment (emilyb7) [inline src/components/Footer.js]: don't forget to turn these into links!

File: src/components/Footer.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+      <section className="w-33 pa2">
+        <div className="flex align-center justify-end">
+          <div>
+          <img className="w4" alt="" src={historicEngland} />
```
Comment (emilyb7) [inline src/components/Footer.js]: why not do some alt text?

File: src/components/Footer.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+          </div>
+          <div>
+          <img className="w4" alt="" src={islington} />
```
Comment (emilyb7) [inline src/components/Footer.js]: and here

File: src/components/About.js [hunk 4]
```diff
@@ -0,0 +1,204 @@
...
+          and conserve assets which contribute to the character, economy and
+          community pride of the borough.</p>
+        </section>
+      </div>
+
+      <div>
+        <section className="mt0 pt4 pb4 bt b--primary bw1 lh-copy">
+          <h2 className="mt0 f4">Project Partners</h2>
+          <p>Our Eyes on Islington is a project in partnership with the
+          &nbsp;<a href="" className="red link">London Borough of Islington</a> and Historic England.</p>
```
Comment (emilyb7) [inline src/components/About.js]: don't forget the link

File: src/components/About.js [hunk 5]
```diff
@@ -0,0 +1,204 @@
...
+
+          <p>In reviewing the Local List, project volunteers applied LB of
+          Islington’s set of heritage criteria. The full list of criteria can
+          be found <a href="" className="red link">here</a>.</p>
```
Comment (emilyb7) [inline src/components/About.js]: and another missing link

**Comment by emilyb7** [comment]
Cool, looks mostly fine but check about the missing links and alt text!

Some other stuff (not specific to the code):

- the main part of the about text looks great on mobile, but the footer looks squashed and social icons overlap the contact details (I'm looking at iphone 6 in the chrome dev tools)
- the individual scrolling panels make it quite hard to scroll down to the footer. Both on the about page and in the app. Have you thought about this?
- Once on the about page the navigation back to the app isn't very clear. I would:
  - make the logo in the header a link
  - change the "About" link in the header to say "home" or "back" once on the about page

File: src/components/About.js (hunk 5 above)
Comment (jessicasalmon) [reply src/components/About.js]: we don't have the links for this text yet but noted this in issue #40 so it doesn't get lost

File: src/components/Footer.js (hunk 1 above)
Comment (jessicasalmon) [reply src/components/Footer.js]: ahh ty :)

**Comment by jessicasalmon** [comment]
Hey @emilyb7, thanks for taking a look at this. I think we've actually done all of your suggested changes in your most recent comment, in future PRs :)

**Pull request merged**